src/main/resources/application.properties

```diff
@@ -17,6 +21,10 @@ quarkus.datasource.password=postgres
 quarkus.datasource.jdbc.url=jdbc:postgresql://127.0.0.1:5432/notifications
 quarkus.datasource.jdbc.max-size=4
 
+%test.quarkus.datasource.username=hook
```

Comment:
Those are also managed by testcontainers

Reply:
This was unused indeed, I removed them.